.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+python:
+    - "2.7"
```

Comment:
Isn't Python 2.x EOL?

Reply:
Yes, Python 2.x is EOL, but I don't think this really matters for now. This is [the example that PlatformIO gives for Travis-CI integration](https://docs.platformio.org/en/latest/integration/ci/travis.html) - so we simply followed that.